src/api/spec/features/webui/comments_spec.rb

```diff
@@ -13,13 +14,10 @@
   end
 
   scenario 'can be answered' do
-    skip_if_bootstrap
     login user
-    comment = create(:comment_project, commentable: Project.first, user: user)
     visit project_show_path(user.home_project)
-    # reply link contains an img element. Thus the regex
-    find('a', text: /.*Reply/).click
-    fill_in "reply_body_#{comment.id}", with: 'Reply Body'
+    find("#reply_link_id_#{comment.id}").click
```

Comment:
And now because we have this spec for both the old and new UIs, we need to use the ID since the old UI has an image and this prevent `click_on('Reply')` from working.